app/web/features/profile/view/UserOverview.tsx

```diff
@@ -43,25 +48,79 @@ const useStyles = makeStyles((theme) => ({
       marginBottom: theme.spacing(0.5),
     },
   },
+
+  cardActions: {
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "stretch",
+    padding: theme.spacing(0.5),
+    "& > *": {
+      margin: theme.spacing(0.5),
+    },
+    "& > :not(:first-child)": {
+      marginLeft: theme.spacing(0.5),
+    },
+  },
 }));
 
-export default function UserOverview({ children }: PropsWithChildren<unknown>) {
+// @todo: move this into /components and decouple it from features/profile because it's used
+//        from the dashboard as well
+export default function UserOverview({
+  actions,
+  showHostAndMeetAvailability = true,
```

Comment:
I see. This is a tricky opinionated topic haha. 

I tend to dislike the `hide...` settings defaulting to false due to seeing DB field names using this pattern, which made the logic harder to read and maintain than it could have been.

For now, I lean towards making it not default to anything, and I'll make it required, so we're explicitly defining if it should show or not. The component is only used in a handful of places, so passing the prop isn't actually much overhead in terms of quantity of code

Reply:
Sounds good to me!

Reply:
Yeah sounds good, at least making it required means we won't have the confusion I described earlier! 👍